pkg/web/api/processor_v1.go

```diff
@@ -34,7 +34,7 @@ type ProcessorOrchestrator interface {
 	// Get will return a single Interface or an error if it doesn't exist.
 	Get(ctx context.Context, id string) (*processor.Instance, error)
 	// Create will make a new Interface.
-	Create(ctx context.Context, name string, parent processor.Parent, cfg processor.Config) (*processor.Instance, error)
+	Create(ctx context.Context, procType string, parent processor.Parent, cfg processor.Config) (*processor.Instance, error)
```

Comment:
Note: this file and the accompanying test file will contain errors once we use the new proto definition because the field `Name` will be renamed to `Type` (just a note that this is expected and will be done in a followup PR).